### PR TITLE
Assignment 2- Julia

### PR DESCRIPTION
**Pull request by manavsinghal157**

### CODE REVIEW

**Comment by shanthanu9** [comment]
You have again submitted Assignment 1 solution in this PR. Can you remove these files?

File: JuliaGPU/session2/Manav/Q1.jl [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            			arr[i],arr[j]=arr[j],arr[i]
+			end
+		end
+        	arr[i+1],arr[high]=arr[high],arr[i+1]
+    		return(i+1)
+	end
+	if low<high
+		pivot=Partition!(arr,low,high)
+    		Quicksort!(arr,low,pivot)
+    		Quicksort!(arr,pivot+1,high)
```
Comment (shanthanu9) [inline JuliaGPU/session2/Manav/Q1.jl:17]: There is an error here in how you recursively call Quicksort!

File: JuliaGPU/session2/Manav/Q3.jl [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+			ans+=inc
+		end
+		ans-=inc
+		inc/=10
+	end
+	return ans
+end
+Arr = rand(2^20)
+ArrCopy = copy(Arr)
+array_sqrt(Arr) = @. Arr = bin_sqrt(Arr)
```
Comment (shanthanu9) [inline JuliaGPU/session2/Manav/Q3.jl:34]:
```suggestion
array_sqrt!(Arr) = @. Arr = bin_sqrt(Arr)
```

**Review by shanthanu9 — CHANGES_REQUESTED** [review]

File: JuliaGPU/session2/Manav/Q3.jl (hunk 2 above)
Comment (manavsinghal157) [reply JuliaGPU/session2/Manav/Q3.jl:34]: Ok Shall change that...

File: JuliaGPU/session2/Manav/Q1.jl (hunk 1 above)
Comment (manavsinghal157) [reply JuliaGPU/session2/Manav/Q1.jl:17]: OK shall resolve it and resend...

**Pull request closed without merging**